test/verify/check-storage-nfs

```diff
@@ -170,6 +182,35 @@ class TestStorageNfs(StorageCase):
         b.click('#dialog [data-field="remote"] button.pf-c-select__toggle-button')
         wait_for_exports()
 
+    def testNfsMountWithoutDiscovery(self):
+        m = self.machine
+        b = self.browser
+
+        self.login_and_go("/storage")
+
+        m.execute("mkdir /home/foo /home/bar")
+        m.write("/etc/exports", "/home/foo 127.0.0.0/24(rw)\n/home/bar 127.0.0.0/24(rw)\n")
+        m.execute("systemctl restart nfs-server")
+
+        # Break showmount.  Cockpit uses showmount to list exported
+        # directories, but that relies on a properly configured
+        # firewall etc.  Even if showmount doesn't work, Cockpit
+        # should allow people to mount arbitrary directories.
+        m.execute("chmod a-x /usr/sbin/showmount")
```

Comment:
This needs to be cleaned up in teardown, so an addCleanup  is necessary here.

Reply:
This is not a `@nondestructive` test, though.

Reply:
Aha! Did not realize, thanks!